src/DSi_SPI_TSC.cpp

```diff
@@ -121,6 +121,11 @@ void DSi_TSC::SetTouchCoords(u16 x, u16 y)
     }
 }
 
+void DSi_TSC::MoveTouchCoords(SPITouchScreenMovement x, SPITouchScreenMovement y)
```

Comment:
Is there a reason that `DSi_TSC`'s override of this method does nothing?

Reply:
Oh, sorry, I forgot to mention. I've only tested the touchscreen camera controls with the DS mode, but not with the DSi mode. Since the implementation of SetTouchCoords was different for each of them, I decided to include that override to avoid unexpected behaviours. At the time I had implemented that only for KH Days, so there was no use in implementing that function either.

I can make some tests with the DSi mode to implement that function, but that may take a little while.